cmd/containerd-nydus-grpc/pkg/command/flags.go

```diff
@@ -7,61 +7,49 @@
 package command
 
 import (
-	"os"
-	"path/filepath"
-	"time"
-
-	"github.com/containerd/containerd/log"
-	"github.com/containerd/nydus-snapshotter/cmd/containerd-nydus-grpc/pkg/logging"
-	"github.com/containerd/nydus-snapshotter/config"
-	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
 	"github.com/urfave/cli/v2"
 )
 
 const (
-	defaultAddress             = "/run/containerd-nydus/containerd-nydus-grpc.sock"
-	defaultLogLevel            = logrus.InfoLevel
-	defaultRootDir             = "/var/lib/containerd-nydus"
-	oldDefaultRootDir          = "/var/lib/containerd-nydus-grpc"
-	defaultGCPeriod            = "24h"
-	defaultPublicKey           = "/signing/nydus-image-signing-public.key"
-	defaultRotateLogMaxSize    = 200 // 200 megabytes
-	defaultRotateLogMaxBackups = 10
-	defaultRotateLogMaxAge     = 0 // days
-	defaultRotateLogLocalTime  = true
-	defaultRotateLogCompress   = true
+	defaultAddress           = "/run/containerd-nydus/containerd-nydus-grpc.sock"
+	defaultLogLevel          = logrus.InfoLevel
+	defaultRootDir           = "/var/lib/containerd-nydus"
+	defaultGCPeriod          = "24h"
+	defaultPublicKey         = "/signing/nydus-image-signing-public.key"
+	DefaultDaemonMode string = "multiple"
+	FsDriverFusedev   string = "fusedev"
 )
 
 type Args struct {
-	Address                  string
-	LogLevel                 string
-	LogDir                   string
-	ConfigPath               string
-	RootDir                  string
-	CacheDir                 string
-	GCPeriod                 string
-	ValidateSignature        bool
-	PublicKeyFile            string
-	ConvertVpcRegistry       bool
-	NydusdBinaryPath         string
-	NydusImageBinaryPath     string
-	SharedDaemon             bool
-	DaemonMode               string
-	FsDriver                 string
-	SyncRemove               bool
-	EnableMetrics            bool
-	MetricsFile              string
-	EnableStargz             bool
-	DisableCacheManager      bool
-	LogToStdout              bool
-	EnableNydusOverlayFS     bool
-	NydusdThreadNum          int
-	CleanupOnClose           bool
-	KubeconfigPath           string
-	EnableKubeconfigKeychain bool
-	RecoverPolicy            string
-	PrintVersion             bool
+	Address                  string `toml:"address"`
+	LogLevel                 string `toml:"log_level"`
+	LogDir                   string `toml:"log_dir"`
+	ConfigPath               string `toml:"config_path"`
+	RootDir                  string `toml:"root_dir"`
+	CacheDir                 string `toml:"cache_dir"`
+	GCPeriod                 string `toml:"gc_period"`
+	ValidateSignature        bool   `toml:"validate_signature"`
+	PublicKeyFile            string `toml:"public_key_file"`
+	ConvertVpcRegistry       bool   `toml:"convert_vpc_registry"`
+	NydusdBinaryPath         string `toml:"nydusd_binary_path"`
+	NydusImageBinaryPath     string `toml:"nydus_image_binary_path"`
+	SharedDaemon             bool   `toml:"shared_daemon"`
```

Comment:
`SharedDaemon` should already be deprecated. It is kept in CLI flags for back compatibility. So we can safely reject it into nydus-snapshotter's configuration file